test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs

```diff
@@ -70,6 +71,35 @@ public void InstallPackageFromUI()
             CommonUtility.AssertPackageInPackagesConfig(VisualStudio, project, "newtonsoft.json", "9.0.1", XunitLogger);
         }
 
+        [StaFact]
+        public void InstallPackageToWebSiteFromUI()
+        {
+            // Arrange
+            EnsureVisualStudioHost();
+            var dte = VisualStudio.Dte;
+            var solutionService = VisualStudio.Get<SolutionService>();
+
+            WebSiteProjectItems websiteProject = new WebSiteProjectItems();
+
+            foreach (WebSiteProjectItem item in websiteProject.InitProjectItems())
```

Comment:
Given that we're testing multiple projects, we should create a `Theory`, and provide the data instead of looping over all the templates.

Here's an example: https://github.com/NuGet/NuGet.Client/blob/d22c6743bf6237c39b98d5d680e06e6c33e97f1c/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NetCoreProjectTestCase.cs#L195-L197

Reply:
Thanks for your suggestion. We are investigating how to fix this problem with `Theory` and fix all the issues together later.